A2rchi/chains/chain.py

```diff
@@ -52,20 +53,32 @@ def __init__(self):
 
         # only run update vectorstore thread if dynamic updating of vectorstore allowed with ssh client
         if self.utils_config["data_manager"]["use_HTTP_chromadb_client"]:
-            update_vectorstore_thread = Thread(target = self.update_vectorstore)
+            update_vectorstore_thread = Thread(target=self.update_vectorstore)
             update_vectorstore_thread.start()
 
+    # def update_vectorstore(self):
+    #     self.dataManager.update_vectorstore()
+    #     self.vectorstore = self.dataManager.fetch_vectorstore()
+    #     self.chain = BaseChain.from_llm(self.llm, self.vectorstore.as_retriever(), return_source_documents=True)
+
+    #     return
+
     def update_vectorstore(self):
         while not self.kill:
             time.sleep(int(self.config["chain_update_time"]))
             self.lock.acquire()
-            self.vectorstore = Chroma(client = self.client, collection_name = self.collection_name,
-                                      embedding_function = self.embedding_model)
-            self.chain = BaseChain.from_llm(self.llm, self.vectorstore.as_retriever(), return_source_documents = True)
-            print(" N Coll: ",self.client.get_collection(self.collection_name).count())
+            self.vectorstore = Chroma(
+                client=self.client,
+                collection_name=self.collection_name,
+                embedding_function=self.embedding_model,
+            )
+            self.chain = BaseChain.from_llm(self.llm, self.vectorstore.as_retriever(), return_source_documents=True)
+            print(f" N Collection: {self.client.get_collection(self.collection_name).count()}")
```

Comment:
This prints out the number of collumns (or entries) in the collection, not the Number of collections as the print statement suggests. We should probably change it to "N entries", "N chunks", or something like it.